web/src/routes/(app)/home/+page.svelte

```diff
@@ -240,4 +242,10 @@
 		background-color: rgb(240, 240, 240);
 		color: var(--accent-gray);
 	}
+
+	@media screen and (min-width: 600px) {
```

Comment:
レスポンシブ対応が [`max-width: 600px`](https://github.com/SnowCait/nostter/blob/89650429ebc7d72272b59210d3888920bc3a1e30/web/src/routes/(app)/%2Blayout.svelte#L198) でレイアウトされているのでちょうど 600px のときに意図しないレイアウトになっていそうです。
```suggestion
	@media screen and (min-width: 601px) {
```